bouncer/shared/send_btc.ts

```diff
@@ -36,7 +36,7 @@ export async function fundAndSendTransaction(
 }
 
 export async function sendVaultTransaction(
-  nulldataUtxo: string,
+  nulldata_payload: string,
```

Comment:
javascript convention
```suggestion
  nulldataPayload: string,
```